lib/game.rb

```diff
@@ -0,0 +1,21 @@
+class Game
+  attr_accessor :player1, :player2, :board, :winning_combination
```

Comment:
- [ ] The student uses attr_readers/attr_writers. Only uses attr_accessors if they really need it and must be aware that by using it they allow writing that property from outside the class. That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

In this case, the winning_combination shouldn't be updated. So you should use attr_reader for it.